rust/src/helpers/array.rs

```diff
@@ -211,30 +202,23 @@ mod tests {
     a = []
     "},
         indoc ! {r"
-    a = [
-    ]
+    a = []
     "}
     )]
     #[case::single(
         indoc ! {r#"
     a = ["A"]
     "#},
         indoc ! {r#"
-    a = [
-      "A",
-    ]
+    a = ["A"]
     "#}
     )]
     #[case::newline_single(
         indoc ! {r#"
-    a = [
-      "A"
-    ]
+    a = ["A"]
     "#},
         indoc ! {r#"
-    a = [
-      "A",
```

Comment:
I consider this regression, because now adding a new line no longer it's diff friendly 🤔

Reply:
If there's no trailing comma, it behaves like black, it's not expanded. There are lots of lists that you don't expect to change much, so you can save 66% space by making small arrays (less than column width) one line using the comma at the end.

But sadly this is only opt-in, normally the column-width is set to 1. I changed the column-width to 80 for this test, otherwise the test would always force to multiline, which also always forces the addition of the final comma.

Reply:
👍

Reply:
(The broken test is a regression, though!)

Reply:
Also, keep in mind this is the sorting step, not the formatting step (the test adds formatting, but it's not actually formatting here normally). It will still go through the formatter later. The idea is that it doesn't change anything in the sorting step, trailing comma status is kept.